### ISSUE TITLE
Use dnf configuration where applicable

### ISSUE THREAD

**Issue by genodeftest** [timlau/yumex-dnf#106]
Citing @Woi in https://github.com/timlau/yumex-dnf/issues/92#issuecomment-215997442:

> yumex-dnf with default settings (that is: no settings changed by user) should behave the same as dnf does.

I second that idea. Shouldn't yumex-dnf only use dnf configuration? By that I mean yumex-dnf should read these settings listed below from dnf config files and store them in dnf config files.

This affects:
- cache expiration interval (metadata_expire=24000 in /etc/dnf/dnf.conf)
- number of installonly packages to keep (installonly_limit=3 in /etc/dnf/dnf.conf)
- erase unused requirements (clean_requirements_on_remove=True in /etc/dnf/dnf.conf)

New options to add:
- install_weak_deps=True in /etc/dnf/dnf.conf
- bandwidth (?)
- deltarpm (?)

**Comment by timlau** [comment]
Agree on using dnf.conf for default, but I dont like updating dnf.conf from yumex-dnf

**Comment by Woi** [comment]
@timlau 

> Agree on using dnf.conf for default, but I dont like updating dnf.conf from yumex-dnf

I totally agree: changing files in `/etc/` by a user application sounds scary.

**Comment by Conan-Kudo** [comment]
@timlau Why not allow for user-override of select options stored in a dot file in the user's home directory? I'm not sure which ones would make sense to allow override or not, but that could be much safer than editing system configs.

**Comment by piapreawhu** [comment]
perhaps this is related (if not, I'll open another issue) -- it would be nice if yumex-dnf made use of any enabled dnf plugins. for example, I recently installed the local plugin and was disappointed to find it only works when i use dnf from the command line, and not when i use yumex-dnf :-(

**Comment by ameeno** [comment]
+1 +1

I have set up my dnf.conf how i like it, yumex should just act as a gui, similar to discover/gnome-software. not sure why it has its own cache/keep/other stuffs.

**Comment by timlau** [comment]
@ameeno I'm working on at total rewrite of Yum Extender, it should not have the same issues 
checkout 
https://github.com/timlau/yumex-ng
